sympy/sets/sets.py

```diff
@@ -1877,10 +1877,10 @@ def _union(self, other):
 
     def _contains(self, other):
         """
-        Tests whether an element, other, is in the set.
+        Tests whether an element 'other' is in the set.
 
         Relies on Python's set class. This tests for object equality
-        All inputs are sympified
+        All inputs are simplified
```

Comment:
I'd leave it as is. `sympify()` and `simplify()` are two explicitly different functions as pointed out in the [tutorial](http://docs.sympy.org/latest/tutorial/basic_operations.html?highlight=sympified#converting-strings-to-sympy-expressions). As such changing it to simplified would be likely to create more confusion as that is not the function that is being used.